<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>attraction</class>
 <widget class="QDialog" name="attraction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_at">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>예약하는곳</string>
   </property>
  </widget>
  <widget class="QPushButton" name="slt_btn">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>360</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>전 환</string>
   </property>
  </widget>
  <widget class="QPushButton" name="out_btn">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>80</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>나가기</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>591</width>
     <height>311</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>581</width>
     <height>261</height>
    </rect>
   </property>
   <property name="verticalHeaderFormat">
    <enum>QCalendarWidget::NoVerticalHeader</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_attraction">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>450</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Spot">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>450</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Spot</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_hotel">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>500</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="hotel">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>500</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Hotel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="take_btn">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>620</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>예약</string>
   </property>
  </widget>
  <widget class="QLabel" name="hotel_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>550</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Guide</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_guide">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>550</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
